lib/OpenQA/WebAPI/Controller/Step.pm

```diff
@@ -253,14 +253,9 @@ sub edit {
     $screenshot->{tags} = $screenshot->{area} = $screenshot->{matches} = [];
     unshift(@needles, $screenshot);
 
-    # stash properties
-    my $properties = {};
-    for my $property (@{$default_needle->{properties}}) {
-        $properties->{$property} = $property;
-    }
     $self->stash('needles',        \@needles);
     $self->stash('tags',           $tags);
-    $self->stash('properties',     $properties);
+    $self->stash('properties',     $default_needle->{properties});
```

Comment:
The previous code took explicit effort to make a copy here. Is this really no longer needed/wanted?

Reply:
if I did not mistake, the previous code was used to convert the default needle's property array to hash. After my modification, the property has been a hash. so I think it is not needed.

Reply:
The properties parameter is used to judge if the default needle has workaround property.  As my new modified, the template does not need to check this (javascript does), so I will remove the properties from there.